source/PlayerInfo.h

```diff
@@ -313,13 +313,18 @@ class PlayerInfo {
 
 	// Keep track of any outfits that you have sold since landing. These will be
 	// available to buy back until you take off.
-	const std::map<const Outfit*, int> &GetStock() const;
-	int Stock(const Outfit *outfit) const;
-	void AddStock(const Outfit *outfit, int count);
+	const std::map<const Outfit*, int> &GetOutfitStock() const;
```

Comment:
style (yes I know it was in the existing code)
```suggestion
	const std::map<const Outfit *, int> &GetOutfitStock() const;
```